src/components/VirtualizedGrid/__tests__/virtualizedGrid.jest.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import VirtualizedGrid from "../VirtualizedGrid";
```

Comment:
We have new standards for our tests files structure. You can see an example in the button group tests files. We separate between the snapshot tests and the user interactions tests
@orrgottlieb Do you think there is other tests which is mandatory we can add here?